src/components/NavBar.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+
+const NavBar = () => {
+  const links = [
+    {
+      id: 1,
+      path: '/',
+      text: 'Calculator',
+    },
+    {
+      id: 2,
+      path: '/home',
+      text: 'Home',
+    },
+    {
+      id: 3,
+      path: '/quote',
+      text: 'Quote',
+    },
+  ];
+
+  return (
+    <div className="navbar-component">
+      <h1>Math Magicians</h1>
+      <nav className="navbar">
+        <ul className="navbar-list">
+          {links.map((link) => (
+            <li key={link.id} className="navbar-list-items">
+              <NavLink to={link.path} activeClassName="active-link" exact>
+                {link.text}
+              </NavLink>
+              <span className="pipe-divisor"> | </span>
```

Comment:
* Kindly make your webpage match the given wireframe in the project requirement the links should be aligned to the right 
* YOURS 
![image](https://user-images.githubusercontent.com/52497246/130824515-c4410e31-487e-41cd-8862-ef718e7627e4.png)
* ORIGINAL 
![image](https://user-images.githubusercontent.com/52497246/130824580-87735f49-d2b9-42e0-8933-ed862af33708.png)

Reply:
Thanks for the review, I have worked on the changes and requested another review